src/config.rs

```diff
@@ -357,6 +365,9 @@ mod tests {
             [assign]
             users_on_vacation = ["jyn514"]
 
+            [review-work-queue]
+            enabled-for-teams = ["aaa", "bbb"]
```

Comment:
I would like to enable this feature only for a subset of teams, rather than tracking all PRs of every contributor from the start.

But that would add a little of complexity to this PR. Maybe worth adding in a subsequent patch, when/if this one is merged?